dev/org.eclipse.codewind.core/src/org/eclipse/codewind/core/internal/CodewindApplication.java

```diff
@@ -262,7 +262,7 @@ public URL getRootUrl() {
 	public URL getMetricsUrl() {
 		try {
 			if ((!this.injectMetrics) && this.metricsAvailable) {
-				return new URL(getBaseUrl(), projectLanguage.getMetricsRoot());
+				return new URL(getRootUrl(), projectLanguage.getMetricsRoot());
```

Comment:
Will this work if the user changes the context root in the .cw_settings file?  The user could change it to anything and that will be reflected in the root url.

Reply:
Locally this works even if the user changes the context root in .cw_settings:

E.g. (my logs):
```
getRootUrl() is: http://127.0.0.1:32799/asdfasdfa
projectLanguage.getMetricsRoot() is: javametrics-dash/?theme=dark
metricsUrl is: http://127.0.0.1:32799/javametrics-dash/?theme=dark
```

So I guess `URL metricsUrl = new URL(getRootUrl(), projectLanguage.getMetricsRoot());` just takes the host from getRootUrl() , ignoring the context root (edited)